components/brave_wallet_ui/utils/balance-utils.ts

```diff
@@ -90,3 +93,7 @@ export function setBalance(
   const chainBalances = accountBalances.chains[chainId]
   chainBalances.tokenBalances[contractAddress.toLowerCase()] = balance
 }
+
+export function CreateEmptyTokenBalancesRegistry() {
```

Comment:
The current convention is to use PascalCase only for constructors. Prefer using `createEmptyTokenBalancesRegistry` here.